our_site/planner/templates/planner/event_home_owned.html

```diff
@@ -1,40 +1,91 @@
-                                {% for role in roles_list %}
-                                    <li class="px-6">  •{{ role.name }} </li>
-                                {% endfor %}
-                            </div>
-                        </ul>
-                    </div>
-                </p>
+    <section class="section" >
+        <div class="container has-background-white p-3">
+            <div class="columns">
+                <div class="column has-text-left">
+                    <form method="post">
+                        {% csrf_token %}
+                        <input class="button is-medium is-warning" type="submit" value="Edit Event" name="Edit Event" id="edit-event">
+                    </form>
+                </div>
             </div>
-        </section>
-
-        <div class="container">
-            <form method="post">
-                {% csrf_token %}
-                <input class="button is-medium is-warning" type="submit" value="Edit Event" name="Edit Event" id="edit-event">
-            </form>
-            <br>
-            <form action="{% url 'planner:dashboard' %}">
-                <input class="button is-medium is-warning" type="submit" value="Back to Dashboard" />
-            </form>
         </div>
-    </div>
 
-{% endblock %}
+        <div class="container has-background-white p-5">
+            <h1 class="title is-1 has-text-centered" style="font-size:65px; color: #7f0000">{{ event.name }}</h1>
+            <div class="content">
+                <ul>
+                    <p style="color:black; font-size:20px"><strong style="font-size:25px; color:#7f0000">Description:</strong> {{ event.description }}</p>
+                    <p style="color:black; font-size:20px"><strong style="font-size:25px; color:#7f0000">Host:</strong> {{ event.owner.user }}</p>
+                    <p style="color:black; font-size:20px"><strong style="font-size:25px; color:#7f0000">Location:</strong> {{ event.location }}</p>
+                    <p style="color:black; font-size:20px"><strong style="font-size:25px; color:#7f0000">Date:</strong> {{ event.date }}</p>
+                </ul>
+            </div>
+            
+            <div class="columns">
+                <div class="column has-text-left box">
+                    <h2 class="subtitle is-4">Attendees:</h2>
+                    <ul>
+                        {% for attendee in attendees %}
+                            <li>{{ attendee.user.username }}</li>
+                        {% endfor %}
+                    </ul>
+                </div>
+                <div class="column has-text-left box">
+                    <h2 class="subtitle is-4">Invitees:</h2>
+                    <ul>
+                        {% for invitee in invitees %}
+                            <li>{{ invitee.user.username }}</li>
+                        {% endfor %}
+                    </ul>
+                </div>
+                <div class="column has-text-left box">
```

Comment:
third box is always larger than other two?? are we okay removing this cause it looks pretty jank with it. Or could we possibly find a way to fix it? I tried looking a little but couldn't figure out what was causing it.